conformance/conformance_ruby.rb

```diff
@@ -1,12 +1,14 @@
-#!/usr/bin/env ruby
-#
 # Protocol Buffers - Google's data interchange format
 # Copyright 2008 Google Inc.  All rights reserved.
 #
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file or at
 # https://developers.google.com/open-source/licenses/bsd
 
+$LOAD_PATH.unshift ENV['PWD']
```

Comment:
I like the `-I` hashbang better :P. Either way we should add a TODO and file an issue with rules_ruby.  The root problem here is that there's no way to strip the `ruby/lib` prefix from the ruby runtime code, so importing `google/protobuf` doesn't work in Bazel.  This will affect any downstream Ruby/Bazel users

Reply:
I changed this because local testing showed that once we changed the `rb_binary` rule to pass a `main` attribute, the hashbang line wasn't used, presumably because the file is getting read by a ruby parser directly rather than a shell. I will add a TODO pointing to the feature request.

Reply:
That's strange.. my local tests definitely read the hashbang line after adding the `main` attribute